dottle

```diff
@@ -1,106 +1,85 @@
 #!/bin/sh
 
-# DEBUG=$(printf "${DEBUG}" | tr '[:upper:]' '[:lower:]')
-
+DEBUG=false
 CONFIG_FILE="${CONFIG_FILE:-install.conf.yml}"
-
 BASEDIR="$(cd "$(dirname "$0")" && pwd)"
-
 cd "${BASEDIR}"
 
-# Declaration of color variables {{{
-ESC_SEQ="\033["
-COL_RESET=$ESC_SEQ"0m"
-COL_BLACK=$ESC_SEQ"0;30;01m"
-COL_RED=$ESC_SEQ"0;31;01m"
-COL_GREEN=$ESC_SEQ"0;32;01m"
-COL_YELLOW=$ESC_SEQ"0;33;01m"
-COL_BLUE=$ESC_SEQ"0;34;01m"
-COL_MAGENTA=$ESC_SEQ"0;35;01m"
-COL_CYAN=$ESC_SEQ"0;36;01m"
-COL_WHITE=$ESC_SEQ"0;37;01m"
-# }}}
-# Special print funcions (`ok`, `warn`, `error`, `debug`) {{{
-ok () {
-    if [ ! -z "$1" ]; then
-        printf "${COL_GREEN}[ok]${COL_RESET} %b\n" "$1"
-    else
-        printf "${COL_GREEN}[ok]${COL_RESET}"
-    fi
-}
-
-warn () {
-    printf "${COL_BLUE}[warning]${COL_RESET} %b\n" "$1"
-}
-
-error () {
-    if [ ! -z "$1" ]; then
-        printf "${COL_RED}[error]${COL_RESET} %b\n" "$1" 1>&2
-    else
-        printf "${COL_RED}[error]${COL_RESET}" 1>&2
-    fi
-}
-
-debug () {
-    if [ -n "$DEBUG" ]; then
-        printf "$COL_MAGENTA[debug]$COL_RESET %b\n" "$1" 1>&2
-    fi
-}
+# color variables
+ESC="\033["
+RESET=$ESC"0m"
+RED=$ESC"0;31;01m"
+GREEN=$ESC"0;32;01m"
+YELLOW=$ESC"0;33;01m"
+BLUE=$ESC"0;34;01m"
+MAGENTA=$ESC"0;35;01m"
 
-running () {
-    printf "$COL_YELLOW[running]$COL_RESET %b" "$1"
+# print funcion
+output () {
```

Comment:
The syntax in this function seems a little backwards to me.

I prefer `output "level" "message"` instead of `output "message" "level"`